vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -176,9 +294,181 @@ void MediaTwitter::SavePublisherInfo(
                           duration,
                           window_id,
                           callback);
+
   if (!media_key.empty()) {
     ledger_->SetMediaPublisherInfo(media_key, publisher_key);
   }
 }
 
+void MediaTwitter::OnSaveMediaVisit(
+    ledger::Result result,
+    ledger::PublisherInfoPtr info) {
+  // TODO(nejczdovc): handle if needed
+}
+
+void MediaTwitter::FetchDataFromUrl(
+    const std::string& url,
+    braveledger_media::FetchDataFromUrlCallback callback) {
+  ledger_->LoadURL(url,
+                   std::vector<std::string>(),
+                   std::string(),
+                   std::string(),
+                   ledger::URL_METHOD::GET,
+                   callback);
+}
+
+void MediaTwitter::OnMediaActivityError(const ledger::VisitData& visit_data,
+                                        uint64_t window_id) {
+  std::string url = TWITTER_TLD;
+  std::string name = TWITTER_MEDIA_TYPE;
+
+  if (!url.empty()) {
```

Comment:
DCHECK

Reply:
Does it even make sense to check if url is empty here? It's assigned to a static definition of `TWITTER_TLD` so it's impossible to be empty...

Reply:
will add DCHECK just for the case if someone accidentally deletes something

Reply:
OK, never hurts to be safe.